src/changelog/linker/mapper/leadingv_test.go

```diff
@@ -0,0 +1,198 @@
+package mapper
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/Masterminds/semver"
+	"github.com/newrelic/release-toolkit/src/changelog"
+	"github.com/newrelic/release-toolkit/src/changelog/linker"
+	"github.com/stretchr/testify/assert"
+)
+
+type mapperMock struct {
+	link string
+}
+
+func (m *mapperMock) Map(dep changelog.Dependency) string {
+	if m.link == "" {
+		return ""
+	}
+	return m.link + "-" + dep.To.Original()
+}
+
+func TestLeadingVCheck_Map(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name        string
+		wrapped     linker.Mapper
+		dep         changelog.Dependency
+		checkValue  string
```

Comment:
I'm struggling a bit to understand this test. What is `checkValue exactly`?

Reply:
It is the link value the test function expects for each of the test cases. 

It is actually a bit confusing. I started defining a check function for each test case and then I ended up using a _common_ check function and a string instead.

I'll try to make it easier to while tackling the rest of the PR comments.

Reply:
I changed it back. Now the `checkLink` function is directly defined and I think the test cases are a bit easier to follow. Changed here: [dc67e07](https://github.com/newrelic/release-toolkit/pull/134/commits/dc67e07f5c7412344599d6161c6708350dfc70a5). Let me know your thoughts.